src/master-playlist-controller.js

```diff
@@ -360,6 +360,11 @@ export class MasterPlaylistController extends videojs.EventTarget {
       this.updateDuration();
 
       if (!updatedPlaylist.endList) {
+        // kick segmentloader back into gear if the playlist isn't
```

Comment:
I believe that is why, but now since the playlist loader has a delay, it should probably be done for both live and vod